internal/service/connections.go

```diff
@@ -113,13 +118,6 @@ func (s *DashNGoImpl) DeleteAllConnections(filter filters.Filter) []string {
 func (s *DashNGoImpl) UploadConnections(filter filters.Filter) []string {
 	var dsListing []models.DataSourceListItemDTO
 
-	//TODO: remove code after next release
```

Comment:
Might be nice to leave this in for one more release? Not critical but deprecations usually aren't that short in my experience. Also we have this error check still elsewhere I believe?

Reply:
I would say it depends on the version, could be around for more minor patch, but a version bump I would say remove it as the enumeration implies a breaking change.